### PR TITLE
Updated airways, airports, fixes, runways, KJFK & KEWR SIDs & STARs

### PR DESCRIPTION
**Pull request by aldobenitez**

### CODE REVIEW

File: Include/XA/kzny/kewr.sid [hunk 1]
```diff
@@ -1,122 +1,126 @@
...
+40.69083154;-74.18138771;<br>;
+MERIT;MERIT;
+40.69083154;-74.18138771;<br>;
+CMK;CMK;
+40.69083154;-74.18138771;<br>;
+GREKI;GREKI;
+40.69083154;-74.18138771;<br>;
+IGN;IGN;
 
 KEWR;04L:04R:11:22L:22R:29;NEWARK;WUDGO;WUDGO;
```
Comment (wessupermare) [inline Include/XA/kzny/kewr.sid:57]: These two SIDs (NEWARK & LIBERTY) seem to be identical except that the NEWARK SID allows more runways. Not sure if this is intentional or an import error. I've tried checking the FAA publications of the SIDs, but the diagrams are too chaotic for me to correlate with Aurora's view. I'll approve it assuming the difference between the SIDs is not related to the routing as displayed on Aurora.

**Review by wessupermare — APPROVED** [review]
Looks good! The KJFK STARs in particular seem to have been cleaned up especially well.

File: Include/XA/kzny/kewr.sid (hunk 1 above)
Comment (aldobenitez) [reply Include/XA/kzny/kewr.sid:57]: Indeed. One SID is only for departures out of 22L/22R while the other is for all runways. It is a vectors only SID but I included it show all accepted exits.

File: Include/XA/kzny/airac/awlow.awl [hunk 2]
```diff
@@ -1,991 +1,927 @@
 //Airway tracks
```
[the remark this answers is not in the document]
Comment (aldobenitez) [reply Include/XA/kzny/airac/awlow.awl:1]: Airways were generated via IAB. It allows to extract airways via FIX names or via coordinates. Can be changed easily. No manually maintenance expected for this.

**Pull request merged**